### PR TITLE
AO3-4309 i18n batch subscription emails

### PR DESCRIPTION
**Pull request by zz9pzza**
https://otwarchive.atlassian.net/browse/AO3-4309

Tests will need to be done for adding a new work, adding a new work back dated and adding a new chapter. Please look at html and plain text.

### CODE REVIEW

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 1]
```diff
@@ -6,11 +6,11 @@
     <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
     <% work_link = style_link(this_work.title.html_safe, work_url(this_work)) %>
 
-    <%= this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe %> posted a
+    <%= this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe %> <%= t 'user_mailer.batch_subscription_notification.posted_a' %> 
     <% if creation.is_a?(Work) %>
-      <%= this_work.backdate ? "backdated" : "new" %> work:
+      <%= if this_work.backdate then t 'user_mailer.batch_subscription_notification.backdated' else t 'user_mailer.batch_subscription_notification.new' end  %> <%= t 'user_mailer.batch_subscription_notification.work' %>
```
Comment (ariana-paris) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: This construct will result in a string that goes adjective-noun -- which will be a bummer for languages where adjectives follow the nouns, or even more fun, for where that will vary depending on what the adjective is (for example, in French the likely translation for "new work" would be constructed "new work" but "backdated work" would go "work backdated")

So these need to be separate strings for "backdated work" and "new work".

File: app/views/user_mailer/batch_subscription_notification.text.erb [hunk 2]
```diff
@@ -3,29 +3,29 @@
 <% @seen_summary = {} %>
 <% @creations.each_with_index do |creation, index| %>
 <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-<%= this_work.pseuds.map{|p| p.byline}.to_sentence.html_safe %> posted a <% if creation.is_a?(Work) then %><%= this_work.backdate ? "backdated" : "new" %> work:<% else %>new chapter of <%= this_work.title %> (<%= this_work.word_count %> words):<% chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe %><% end %>
+<%= this_work.pseuds.map{|p| p.byline}.to_sentence.html_safe %> <%= t 'user_mailer.batch_subscription_notification.posted_a' %> <% if creation.is_a?(Work) then %><%= if this_work.backdate then t 'user_mailer.batch_subscription_notification.backdated' else t 'user_mailer.batch_subscription_notification.new' end %> <%= t 'user_mailer.batch_subscription_notification.work' %> <% else %> <%= t 'user_mailer.batch_subscription_notification.new_chapter_of' %> <%= this_work.title %> (<%= this_work.word_count %> words):<% chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe %><% end %>
```
Comment (ariana-paris) [inline app/views/user_mailer/batch_subscription_notification.text.erb]: Aside from the fact this blob of code needs to be split into separate lines to make it easier to read, this has the same problem with "new work" and "backdated work" potentially needing a different syntax.

Comment (zz9pzza) [inline app/views/user_mailer/batch_subscription_notification.text.erb]: I hate the way it is all on one line but I think that is to do with it being a txt template, blank lines will have a tendency to come out on the output. I will try.

**Comment by zz9pzza** [comment]
It looks if a new chapter can be backdated, but we don't have code in to change the message about that should I extend this with backdated chapter support ?

**Comment by cesy** [comment]
This one has fresh conflicts that will need fixing.

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 3]
```diff
@@ -3,51 +3,50 @@
   <% @seen_summary = {} %>
   <% @creations.each_with_index do |creation, index| %>
 
-    <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-    <% work_link = style_link(this_work.title.html_safe, work_url(this_work)) %>
+<% this_work = creation.respond_to?(:work) ? creation.work : creation %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: Since this pull request needs to be updated before it can be merged, a few comments! The indenting has gone a bit wonky here from here down to `<% unless @seen[this_work.id] %>`. I believe what you want is:

```
    <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
    <% pseud = this_work.pseuds.map{|p| p.name}.to_sentence %>
    <% byline = this_work.pseuds.map{|p| p.byline}.to_sentence %>
    <% title = this_work.title.html_safe %>
    <%
      if creation.is_a?(Work) 
        link = style_link(this_work.title.html_safe,work_url(this_work))
        word_count = this_work.word_count
      else
        link = style_link(this_work.title.html_safe,work_chapter_url(this_work, creation))
        chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe
      end
    %>
    <% if creation.is_a?(Work) %>
      <%= 
        if this_work.backdate
          t('user_mailer.batch_subscription_notification.html.work_backdated', byline: byline, title: title, link: link, word_count: word_count).html_safe
        else
          t('user_mailer.batch_subscription_notification.html.work_new', byline: byline, title: title, link: link, word_count: word_count).html_safe
        end
      %>
    <% else %>
      <%= t('user_mailer.batch_subscription_notification.html.new_chapter_of', byline: byline, title: title, link: link, word_count: creation.word_count, chapter_title: chapter_title).html_safe %>
    <% end %>
    <%= t('user_mailer.batch_subscription_notification.html.collected_by', pseud: this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe).html_safe %>

    <% if !creation.summary.blank? && creation.is_a?(Chapter) %>
      <%= t 'user_mailer.batch_subscription_notification.chapter_summary'%> <%= raw to_plain_text(sanitize_field(creation, :summary)) %>
    <% end %>
```

There are various other fixes in what I just pasted, but I'm going to point them out below as well, so you can see where they were.

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 4]
```diff
@@ -3,51 +3,50 @@
...
+<% pseud = this_work.pseuds.map{|p| p.name}.to_sentence %>
+<% byline = this_work.pseuds.map{|p| p.byline}.to_sentence %>
+<% title = this_work.title.html_safe %>
+<% if creation.is_a?(Work) then
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: We don't use `then` in Ruby. In fact, this is a lot of Ruby for a view file -- it would really help readability if this big block were a helper method in, say, mailer_helper.rb, rather than all being here in the view file.

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 5]
```diff
@@ -3,51 +3,50 @@
...
+     link  = style_link(this_work.title.html_safe,work_url(this_work))
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: There should only be one space before the `=`

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 6]
```diff
@@ -3,51 +3,50 @@
...
+     word_count = this_work.word_count
+   else
+     link  = style_link(this_work.title.html_safe,work_chapter_url(this_work, creation))
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: There should only be one space before the `=`

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 7]
```diff
@@ -3,51 +3,50 @@
...
+     chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe
+   end %>
+<% if creation.is_a?(Work) %>
+<%= if this_work.backdate then
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: Again, no `then` in Ruby

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 8]
```diff
@@ -3,51 +3,50 @@
...
+t('user_mailer.batch_subscription_notification.html.work_backdated', byline: byline, title: title, link: link, word_count: word_count ).html_safe
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: There's a stray space between `word_count` and the closing `)`

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 9]
```diff
@@ -3,51 +3,50 @@
...
+else
+t('user_mailer.batch_subscription_notification.html.work_new', byline: byline, title: title, link: link, word_count: word_count).html_safe
+end %>
+<% else %>
+<%= t('user_mailer.batch_subscription_notification.html.new_chapter_of',byline: byline, title: title, link: link, word_count: creation.word_count, chapter_title: chapter_title).html_safe %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: There's a space missing between the `,` and `byline: byline`

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 10]
```diff
@@ -3,51 +3,50 @@
...
+<%end%>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: Should be `<% end %>`

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 11]
```diff
@@ -3,51 +3,50 @@
...
+<%= t( 'user_mailer.batch_subscription_notification.html.collected_by', pseud: this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe).html_safe %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: There's an extra space between `(` and `'user_mailer.batch_subscription_notification.html.collected_by'`.

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 12]
```diff
@@ -3,51 +3,50 @@
...
-      </i> (<%= creation.word_count%> words)
-
-      <% unless @seen[this_work.id] %>
-        <br>
-
-        by <%= this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe %>
-      <% end %>
-    </p>
+<% if !creation.summary.blank? && creation.is_a?(Chapter) %>
+<%= t 'user_mailer.batch_subscription_notification.chapter_summary'%> <%= raw to_plain_text(sanitize_field(creation, :summary)) %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: There's a space missing between `'user_mailer.batch_subscription_notification.chapter_summary'` and the `%>`

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 13]
```diff
@@ -71,5 +70,5 @@
 
 
 <% content_for :footer_note do %>
-  You're receiving this email because you've subscribed to <%= style_footer_link(@subscription.name, polymorphic_url(@subscription.subscribable)) %>. Follow the link to unsubscribe if you no longer wish to receive updates.
+<%=t( 'user_mailer.batch_subscription_notification.html.footer', who: style_footer_link(@subscription.name, polymorphic_url(@subscription.subscribable))).html_safe %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: This should be indented two spaces, like it used to be.

File: app/views/user_mailer/batch_subscription_notification.text.erb [hunk 14]
```diff
@@ -3,29 +3,44 @@
...
 <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-<%= this_work.pseuds.map{|p| p.byline}.to_sentence.html_safe %> posted a <% if creation.is_a?(Work) then %><%= this_work.backdate ? "backdated" : "new" %> work:<% else %>new chapter of <%= this_work.title %> (<%= this_work.word_count %> words):<% chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe %><% end %>
-<%= creation.is_a?(Work) ? work_url(this_work) : work_chapter_url(this_work, creation) %>
-
-<%= creation.is_a?(Work) ? this_work.title.html_safe : chapter_title %> (<%= creation.word_count%> words)<% unless @seen[this_work.id] %>
-by <%= this_work.pseuds.map{|p| text_pseud(p)}.to_sentence.html_safe %><% end %>
+<% pseud = this_work.pseuds.map{|p| p.name}.to_sentence %>
+<% byline = this_work.pseuds.map{|p| p.byline}.to_sentence %>
+<% title = this_work.title %>
+<% if creation.is_a?(Work) then
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.text.erb]: This might be neater written as:

```
<%
  if creation.is_a?(Work)
    link = work_url(this_work)
    word_count = this_work.word_count
  else
    link = work_chapter_url(this_work, creation)
    chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe
  end
%>
```

I've also removed the `then` and the extra spaces between `link` and `=` 

Ideally, though, this should be in a helper method, not the view file.

File: app/views/user_mailer/batch_subscription_notification.text.erb [hunk 15]
```diff
@@ -3,29 +3,44 @@
...
+     link  = work_url(this_work)
+     word_count = this_work.word_count
+   else
+     link  = work_chapter_url(this_work, creation)
+     chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe
+   end %>
+<% if creation.is_a?(Work) %>
+<%= if this_work.backdate then
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.text.erb]: `then` should be removed, as should the extra space between `word_count` and `)` on line 18. This might also be better in a helper or broken up like:

```
<%= 
  if this_work.backdate
    t('user_mailer.batch_subscription_notification.text.work_backdated', byline: byline, title: title, link: link, word_count: word_count)
  else
    t('user_mailer.batch_subscription_notification.text.work_new', byline: byline, title: title, link: link, word_count: word_count)
  end
%>
```

File: app/views/user_mailer/batch_subscription_notification.text.erb [hunk 16]
```diff
@@ -3,29 +3,44 @@
...
+t('user_mailer.batch_subscription_notification.text.work_backdated', byline: byline, title: title, link: link, word_count: word_count )
+else
+t('user_mailer.batch_subscription_notification.text.work_new', byline: byline, title: title, link: link, word_count: word_count)
+end %>
+<% else %>
+<%= t 'user_mailer.batch_subscription_notification.text.new_chapter_of',byline: byline, title: title, link: link, word_count: creation.word_count, chapter_title: chapter_title %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.text.erb]: `'user_mailer.batch_subscription_notification.text.new_chapter_of',byline: byline,` needs a space after the comma

File: app/views/user_mailer/batch_subscription_notification.text.erb [hunk 17]
```diff
@@ -3,29 +3,44 @@
...
+<%end%>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.text.erb]: Should be `<% end %>`

File: app/views/user_mailer/batch_subscription_notification.text.erb [hunk 18]
```diff
@@ -3,29 +3,44 @@
...
+<%= t 'user_mailer.batch_subscription_notification.text.collected_by', pseud: this_work.pseuds.map{|p| text_pseud(p)}.to_sentence.html_safe %>
 
 <% if !creation.summary.blank? && creation.is_a?(Chapter) %>
-Chapter Summary: <%= raw to_plain_text(sanitize_field(creation, :summary)) %>
+<%= t 'user_mailer.batch_subscription_notification.chapter_summary'%> <%= raw to_plain_text(sanitize_field(creation, :summary)) %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.text.erb]: There's a space missing before the `%>`

File: app/views/user_mailer/batch_subscription_notification.text.erb [hunk 19]
```diff
@@ -3,29 +3,44 @@
...
-Warnings: <%= this_work.warning_string %>
-<% if this_work.relationship_string && !this_work.relationship_string.blank? then %>Relationships: <%= this_work.relationship_string %><% end %>
-<% if this_work.character_string && !this_work.character_string.blank? then %>Characters: <%= this_work.character_string %><% end %>
-<% if this_work.freeform_string && !this_work.freeform_string.blank? %>Additional Tags: <%= this_work.freeform_string %><% end %>
-<% if this_work.series.count > 0 %>Series: <%=raw to_plain_text(series_list_for_feeds(this_work)) %><% end %>
+<%= t 'user_mailer.chapters' %> <%= this_work.chapter_total_display %>
+<%= t 'user_mailer.fandom' %> <%= this_work.fandoms.map{|f| f.name}.to_sentence %>
+<%= t 'user_mailer.rating' %> <%= this_work.rating_string %>
+<%= t 'user_mailer.warning' %> <%= this_work.warning_string %>
+<% if this_work.relationship_string && !this_work.relationship_string.blank? then %><%= t 'user_mailer.relationships' %> <%= this_work.relationship_string %><% end %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.text.erb]: I know then `then` was there already, but it shouldn't be

File: app/views/user_mailer/batch_subscription_notification.text.erb [hunk 20]
```diff
@@ -3,29 +3,44 @@
...
+<% if this_work.character_string && !this_work.character_string.blank? then %><%=t 'user_mailer.characters'%> <%= this_work.character_string %><% end %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.text.erb]: The `then` should be removed, and there should be a spaces added here, like so: `<%= t 'user_mailer.characters '%>`

File: app/views/user_mailer/batch_subscription_notification.text.erb [hunk 21]
```diff
@@ -3,29 +3,44 @@
...
+<% if this_work.freeform_string && !this_work.freeform_string.blank? %><%=t 'user_mailer.additional_tags'%> <%= this_work.freeform_string %><% end %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.text.erb]: Needs spaces like: `<%= t 'user_mailer.additional_tags' %>`

File: app/views/user_mailer/batch_subscription_notification.text.erb [hunk 22]
```diff
@@ -3,29 +3,44 @@
...
+<% if this_work.series.count > 0 %><%=t 'user_mailer.series'%> <%=raw to_plain_text(series_list_for_feeds(this_work)) %><% end %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.text.erb]: Needs spaces: `<%= t 'user_mailer.series' %>` and `<%= raw`

File: app/views/user_mailer/batch_subscription_notification.text.erb [hunk 23]
```diff
@@ -33,5 +48,5 @@ Summary:
 <% end %><% end %><% end %>
 
 <% content_for :footer_note do %>
-You're receiving this email because you've subscribed to <%= @subscription.name %>. Follow the link to unsubscribe if you no longer wish to receive updates: <%= polymorphic_url(@subscription.subscribable) %>
+<%=t 'user_mailer.batch_subscription_notification.text.footer', who: @subscription.name, url: polymorphic_url(@subscription.subscribable) %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.text.erb]: Needs a space: `<%= t 'user_mailer.batch_subscription_notification.text.footer'`

File: config/locales/en.yml [hunk 24]
```diff
@@ -42,3 +59,23 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    batch_subscription_notification:
+      subject_linker: "and %{extra} more"
+      chapter_summary: "Chapter Summary: "
```
Comment (sarken) [inline config/locales/en.yml]: Is the space after the colon intentional in `"Chapter Summary: "`? I ask because the rest of the strings don't seem to have one.

File: config/locales/en.yml [hunk 25]
```diff
@@ -42,3 +59,23 @@ en:
           friend at %{invitation_page}.
           Cheers, 
           %{archive_name}
+    batch_subscription_notification:
+      subject_linker: "and %{extra} more"
+      chapter_summary: "Chapter Summary: " 
+      summary: "summary:"
```
Comment (sarken) [inline config/locales/en.yml]: I suspect this should be capitalized.

**Comment by sarken** [comment]
Marking this needs coder action again because I suspect you'll want to update this in light or recent discussion. If not, let me know!

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 26]
```diff
@@ -4,50 +4,51 @@
...
     <% this_work = creation.respond_to?(:work) ? creation.work : creation %>
-    <% work_link = style_link(this_work.title.html_safe, work_url(this_work)) %>
-
-    <%= this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe %> posted a
+    <% pseud = this_work.pseuds.map{|p| p.name}.to_sentence %>
+    <% byline = this_work.pseuds.map{|p| p.byline}.to_sentence %>
+    <% title = this_work.title.html_safe %>
+    <%
+      if creation.is_a?(Work) 
+        link = style_link(this_work.title.html_safe,work_url(this_work))
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: There should be a space after the comma.

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 27]
```diff
@@ -4,50 +4,51 @@
...
+        word_count = this_work.word_count
+      else
+        link = style_link(this_work.title.html_safe,work_chapter_url(this_work, creation))
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: There should be a space after the comma here: `this_work.title.html_safe,work_chapter_url`

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 28]
```diff
@@ -4,50 +4,51 @@
...
-      <% unless @seen[this_work.id] %>
-        <br>
-
-        by <%= this_work.pseuds.map{|p| style_pseud_link(p)}.to_sentence.html_safe %>
-      <% end %>
-    </p>
-
-    <% if creation.summary.present? && creation.is_a?(Chapter) %>
-      <%= style_bold("Chapter Summary:") %> <%= style_quote(raw strip_tags(sanitize_field(creation, :summary))) %>
+    <% if !creation.summary.blank? && creation.is_a?(Chapter) %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: Why the change from `creation.summary.present?` to `!creation.summary.blank?`?

File: app/views/user_mailer/batch_subscription_notification.html.erb [hunk 29]
```diff
@@ -4,50 +4,51 @@
...
+      <%= t '.chapter_summary'%> <%= raw to_plain_text(sanitize_field(creation, :summary)) %>
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.html.erb]: There's a space missing before the `%>`.

Also, I think you might've accidentally put the plain text code in the HTML file here -- the previous HTML version had some styling and stripped tags, but didn't turn it to plain text: `<%= style_quote(raw strip_tags(sanitize_field(creation, :summary))) %>`

File: app/views/user_mailer/batch_subscription_notification.text.erb [hunk 30]
```diff
@@ -3,35 +3,54 @@
...
+    word_count = this_work.word_count
+  else
+    link = work_chapter_url(this_work, creation)
+    chapter_title = (creation.chapter_header + (creation.title.blank? ? "" : ": #{creation.title}")).html_safe
+  end
+%>
+<% if creation.is_a?(Work) %>
+  <%=
+    if this_work.backdate
+     t('.text.work_backdated', byline: byline, title: title, link: link, word_count: word_count )
```
Comment (sarken) [inline app/views/user_mailer/batch_subscription_notification.text.erb]: There's an extra space before the closing `)`

File: config/locales/en.yml [hunk 31]
```diff
@@ -53,6 +70,25 @@ en:
...
+    batch_subscription_notification:
+      subject_linker: "and %{extra} more"
+      chapter_summary: "Chapter Summary:" 
+      text:
+        footer: "You're receiving this email because you've subscribed to %{who}. Follow the link to unsubscribe if you no longer wish to receive updates: %{url}"
+        collected_by: "by %{pseud}"
+        new_chapter_of: "%{byline} posted a new chapter of %{title}:\n%{link}\n\n\
+            %{chapter_title} (%{word_count} words)"
+        work_backdated: "%{byline} backdated a new work:\n%{link}\n\n\
+            %{title} ((%{word_count} words)"
```
Comment (sarken) [inline config/locales/en.yml]: I think there's an extra `(` around the word count here.

File: config/locales/en.yml [hunk 32]
```diff
@@ -53,6 +70,25 @@ en:
...
+        work_new: "%{byline} posted a new work:\n%{link}\n\n\
+            %{title} (%{word_count} words)"
+      html:
+        footer: "You're receiving this email because you've subscribed to %{who}. Follow the link to unsubscribe if you no longer wish to receive updates."
+        work_new: "<p>%{byline} posted a new work:<br><b></i>%{link}</i></b></p><p>%{title} (%{word_count} words)</p>"
```
Comment (sarken) [inline config/locales/en.yml]: Instead of putting the bold and italics around the work title (`%{link}`) here, let's take the same approach we did in delete_work_notification: `<% pretty_title = "<i>#{style_bold(@work.title.html_safe)}</i>".html_safe %>` 

That will cut down on the potential for HTML-related errors in the translation process, e.g. the mistake on the following line, where it currently says `</i>%{link}</i>`.

For reference: https://github.com/otwcode/otwarchive/blob/1fd2e9040e430ee6453530e1ef9ed69a47df35f6/app/views/user_mailer/delete_work_notification.html.erb#L2

**Comment by redsummernight** [comment]
This has conflicts now.

**Comment by zz9pzza** [comment]
Noting that this moves from fragment caching to raw caching, you can do fragment caching with i18n but I am not sure it is needed with the raw caching.

**Comment by sarken** [comment]
Just a heads up that this one might be better left until we resolve a couple issues with the non-internationalized version:

* https://otwarchive.atlassian.net/browse/AO3-5804
* https://otwarchive.atlassian.net/browse/AO3-5805
